test/jobs/parity_check/parity_NSE_example.py

```diff
@@ -0,0 +1,64 @@
+import pandas as pd
+import numpy as np
+import hydroeval as he
+import math 
+# import matplotlib.pyplot as plt
+
+s1 = pd.read_pickle("/glade/u/home/jhreha/t-route/test/jobs/parity_check/chanobs_output/example1")
+s2 = pd.read_pickle("/glade/u/home/jhreha/t-route/test/jobs/parity_check/chanobs_output/example2")
+
+s1_q = s1.iloc[:,::3].sort_index()
+s2_q = s2.iloc[:,::3].sort_index()
+
+print("\n")
+print("****************************************")
+print("****** Are simulated flows equal? ******")
+print("****************************************")
+print("\n")
+test = np.array_equal(s1_q.to_numpy().astype(float), s2_q.to_numpy().astype(float), equal_nan = True)
+
+
+if test:
+    print("YES! - Simulations produce the same results")
+    
+else:
+    print("NO. - Simulations produce different results")
+    diff = abs(s1_q - s2_q)
+    col_idx = diff.max().argmax()
+    row_idx = diff.iloc[:,col_idx].argmax()
+    
+    print("The largest flow difference in flow is: ", diff.max().max(), "cms")
+    print("at link ID", s1_q.index[col_idx], "and timestep", row_idx)
+    print("simulation 1 reports a flow value of", s1_q.iloc[row_idx, col_idx])
+    print("simulation 2 reports a flow value of", s2_q.iloc[row_idx, col_idx])
+    print("\n")
+       
+    print("****************************************")
+    print("**** Are simulated flows similar? ******")
+    print(" ** (... within 0.1% of one another) ***")
+    print("****************************************")

```

Comment:
I suggest that we use NSE as our closeness assessment. % diffs aren't terribly helpful, as I have come to see.